esmvaltool/recipes/examples/recipe_python.yml

```diff
@@ -64,6 +64,7 @@ diagnostics:
     scripts:
       script1:
         script: examples/diagnostic.py
+        write_netcdf: true
```

Comment:
I agree with @bettina-gier that we should not encourage user's to use an option that we deprecated.

Reply:
Thanks, I removed it.